tests/integration/test_interaction_assessment.py

```diff
@@ -517,14 +517,20 @@ def test_popup_visibility(self):
         self.assertFocused(continue_button)
 
     def test_popup_content(self):
-        popup_content = self._page.find_element_by_css_selector('.instructions p:last-child')
-        self.assertEqual("You can review answers / resubmit 5 times", popup_content.text)
+        if (self.MAX_ATTEMPTS and self.MAX_ATTEMPTS > 0):
+            popup_content = self._page.find_element_by_css_selector('.instructions p:last-child')
+            self.assertEqual("You can review answers / resubmit 5 times", popup_content.text)
+        else:
+            self.assertRaises(NoSuchElementException,
+                              self._page.find_element_by_css_selector,
+                              '.instructions p:last-child')
```

Comment:
This method will only be called with ``self.MAX_ATTEMPTS == 5`` so this if/else is unnecessary.